src/Platform/Microsoft.Testing.Platform/OutputDevice/Console/ConsoleLogger.cs

```diff
@@ -0,0 +1,702 @@
+        {
+            LoggerOutcome.Fail or LoggerOutcome.Error => PlatformResources.FailedLowercase,
+            LoggerOutcome.Skipped => PlatformResources.SkippedLowercase,
+            LoggerOutcome.Cancelled or LoggerOutcome.Timeout => $"{PlatformResources.FailedLowercase} ({PlatformResources.CancelledLowercase})",
+            LoggerOutcome.Passed => PlatformResources.PassedLowercase,
+            _ => throw new NotImplementedException(),
+        };
+
+        terminal.SetColor(color);
+        terminal.Append(outcomeText);
+        terminal.ResetColor();
+        terminal.Append(' ');
+        terminal.Append(displayName);
+        terminal.SetColor(TerminalColor.Gray);
+        terminal.Append(' ');
+        terminal.Append($"({duration.TotalSeconds:F1}s)");
+        if (_options.ShowAssembly)
+        {
+            terminal.AppendLine();
+            terminal.Append(SingleIndentation);
+            terminal.Append("from ");
+            AppendAssemblyLinkTargetFrameworkAndArchitecture(terminal, assembly, targetFramework, architecture);
+        }
+
+        terminal.AppendLine();
+
+        FormatErrorMessage(terminal, errorMessage);
+        FormatExpectedAndActual(terminal, expected, actual);
+        FormatStackTrace(terminal, errorStackTrace);
+    }
+
+    private static void AppendAssemblyLinkTargetFrameworkAndArchitecture(ITerminal terminal, string assembly, string? targetFramework, string? architecture)
+    {
+        terminal.AppendLink(assembly, lineNumber: null);
+        if (!RoslynString.IsNullOrWhiteSpace(architecture) && !RoslynString.IsNullOrWhiteSpace(targetFramework))
+        {
+            terminal.Append(" (");
+            terminal.Append(targetFramework);
+            terminal.Append('|');
+            terminal.Append(architecture);
+            terminal.Append(')');
+        }
+    }
+
+    private static void FormatStackTrace(ITerminal terminal, string? errorStackTrace)
+    {
+        if (RoslynString.IsNullOrWhiteSpace(errorStackTrace))
+        {
+            return;
+        }
+
+        terminal.SetColor(TerminalColor.DarkRed);
+        terminal.Append(SingleIndentation);
+        terminal.Append(PlatformResources.StackTrace);
+        terminal.AppendLine(":");
+
+        if (!errorStackTrace.Contains('\n'))
+        {
+            AppendStackFrame(terminal, errorStackTrace);
+            return;
+        }
+
+        string[] lines = errorStackTrace.Split(NewLineStrings, StringSplitOptions.None);
+        foreach (string line in lines)
+        {
+            AppendStackFrame(terminal, line);
+        }
+
+        terminal.ResetColor();
+    }
+
+    private static void AppendStackFrame(ITerminal terminal, string stackTraceLine)
+    {
+        terminal.Append(DoubleIndentation);
+        Match match = GetFrameRegex().Match(stackTraceLine);
+        if (match.Success)
+        {
+            terminal.SetColor(TerminalColor.Gray);
+            terminal.Append("at ");
```

Comment:
Resx?

Reply:
yes. even though stack traces are not localized in .net "core"